Makefile

```diff
@@ -11,8 +11,6 @@
 #  * ENABLE_PFCWD_ON_START: Enable PFC Watchdog (PFCWD) on server-facing ports
 #  * by default for TOR switch.
 #  * SONIC_ENABLE_SYNCD_RPC: Enables rpc-based syncd builds.
-#  * USERNAME: Desired username -- default at rules/config
-#  * PASSWORD: Desired password -- default at rules/config
```

Comment:
>PASSWORD [](start = 5, length = 8)

Could you make the behavior backward compatible. In some case, we are ok to have them in command line. You may add warning for this case.
```
make PASSWORD=$SONIC_PASSWORD target/sonic-broadcom.bin
```

Reply:
Keeping compatibility is good idea so I leave rules/config in Makefile format as well as opportunity to supply USERNAME/PASSWORD via command line.

Warning message is printed to stderr when passing these variables as make(1) parameters
because they will be visible to other users on build host.

Passing USERNAME and PASSWORD from Makefile to build_debian.sh now done via process environment: /proc/<pid>/environ only available to build user: this has nearly same security level as passing it via configuration file (password will be in process memory anyway, either in environment block or in IO buffers when read from config file).